### PR TITLE
Feat/1 checklist

### PR DESCRIPTION
**Pull request by netlight-johannadagfalk**
*Task 11 and 12 combined*

Regarding tests:

bin/guard
bin/rspec
bin/rubocop

--> gives the same results as for dev branch

Same console error as for dev exists: POST https://api-iam.intercom.io/messenger/web/ping 403

Earlier commits are from junk-branch

### CODE REVIEW

File: app/assets/main/react_components/Button.jsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default class Button extends Component {
     const { status, className, children, onClick, href } = this.props;
 
     if (href) {
-      return(
+      return (
```
Comment (netlightebba) [inline app/assets/main/react_components/Button.jsx:9]: Avoid adding extra space, check git Working Tree to see these changes and go through and revert accidental/unnecessary spaces and indents.

File: app/assets/main/react_components/Button.jsx [hunk 2]
```diff
@@ -27,8 +27,8 @@ export default class Button extends Component {
         )
       }
 
-      return(
-        <button 
+      return (
+        <button
```
Comment (netlightebba) [inline app/assets/main/react_components/Button.jsx:31]: Avoid adding extra space, check git Working Tree to see these changes and go through and revert accidental/unnecessary spaces and indents.

File: app/assets/main/react_components/User/ActionFeature.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            id: 6,
+            text: 'Veggie month!',
+            category: 'Food',
+            emoji: '🥗',
+            accpeted: false,
+
+        },
+    ])
+
+    const ifChecked = (id) => {
```
Comment (netlightebba) [inline app/assets/main/react_components/User/ActionFeature.jsx:55]: Maybe name this function _onCheck_? Similar to a button with "onClick".

File: app/assets/main/react_components/User/ActionFeature.jsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+            <div className="max-w-5xl mx-auto space-y-3 t:bg-white t:rounded-lg t:shadow-lg t:p-8 t:border t:border-gray-tint-2">
+                <ChecklistHeader />
+
+                <div class="flex flex-col t:flex-row t:space-x-8 space-y-6 t:space-y-0">
+                    <div class="t:w-1/2 space-y-6">
+                        <Checklist challenges={challenges} ifChecked={ifChecked} isOnlyAccepted={false} />
+                    </div>
+                    <div class="t:w-1/2 space-y-6">
+
+                        <Checklist challenges={challenges} ifChecked={ifChecked} isOnlyAccepted={true} />
```
Comment (netlightebba) [inline app/assets/main/react_components/User/ActionFeature.jsx:70]: Maybe rename prop "ifChecked" on checklist to onCheck? Similar to a button with "onClick".

**Review by netlightebba — CHANGES_REQUESTED** [review]
Good work! I have some thoughts or suggestions, see my comments :D

**Review by netlightebba — APPROVED** [review]

**Review by netlight-saraandersson — APPROVED** [review]

**Pull request merged**